Description: systemd: Use /usr/sbin
 ntpd is installed to /usr/sbin by default.
Forwarded: https://gitlab.com/NTPsec/ntpsec/merge_requests/587
Applied-Upstream: https://gitlab.com/NTPsec/ntpsec/commit/aa7efef660fd8dca2dc20f07c558f6099c709df3
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-26
--- a/etc/ntpd.service
+++ b/etc/ntpd.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=Network Time Service
-Documentation=man:ntpd(1)
+Documentation=man:ntpd(8)
 Wants=network.target
 ConditionVirtualization=!container
 ConditionCapability=CAP_SYS_TIME
@@ -10,7 +10,7 @@
 [Service]
 Type=forking
 PrivateTmp=true
-ExecStart=/usr/bin/ntpd -g -N -u ntp:ntp
+ExecStart=/usr/sbin/ntpd -g -N -u ntp:ntp
 Restart=always
 
 [Install]
